akka-http/src/main/scala/akka/http/javadsl/server/directives/DebuggingDirectives.scala

```diff
@@ -34,7 +34,7 @@ abstract class DebuggingDirectives extends CookieDirectives {
    * @param level One of the log levels defined in akka.event.Logging
    */
   def logRequest(marker: String, level: LogLevel, inner: Supplier[Route]): Route = RouteAdapter {
-    D.logRequest(marker, level) { inner.get.delegate }
+    D.logRequest(marker -> level) { inner.get.delegate }
```

Comment:
`->` doesn't really make sense here, as it's not a mapping - tuple notation might be clearer?
```suggestion
    D.logRequest((marker, level)) { inner.get.delegate }
```